contrib/enfuseAdvanced.lua

```diff
@@ -54,548 +54,146 @@ Same as other export modules
-    value = dt.preferences.read("enfuseAdvanced", "def_save_masks", "bool"),
-    tooltip =_('Save the generated weight masks to your home directory,\nenblend saves masks as 8 bit grayscale, \ni.e. single channel images. \nfor accuracy we recommend to choose a lossless format.'),  
-    reset_callback = function(self) 
-       self.value = dt.preferences.read("enfuseAdvanced", "def_save_masks", "bool")
-    end 
-}
-enf_chkbtn_cpytags = dt.new_widget("check_button"){
-    label = '  Copy Tags', 
-    value = pref_cpytags,
-    tooltip ='Copy tags from first image.',  
-}
-enf_sldr_exposure_weight = dt.new_widget("slider"){
-  label = _('exposure weight'),
-  tooltip = _('set the relative weight of the well-exposedness criterion \nas defined by the chosen exposure weight function. \nincreasing this weight relative to the others will\n make well-exposed pixels contribute more to\n the final output. \ndefault: (1.0)'),
-  hard_min = 0,
-  hard_max = 1,
-  value = dt.preferences.read("enfuseAdvanced", "def_exposure_weight", "float") --1 def
-}
-enf_sldr_saturation_weight = dt.new_widget("slider"){
-  label = _('saturation weight'),
-  tooltip = _('set the relative weight of high-saturation pixels. \nincreasing this weight makes pixels with high \nsaturation contribute more to the final output. \ndefault: (0.2)'),
-  hard_min = 0,
-  hard_max = 1,
-  value = dt.preferences.read("enfuseAdvanced", "def_saturation_weight", "float"), --0.5
-}
-enf_sldr_contrast_weight = dt.new_widget("slider"){
-  label = _('contrast weight'),
-  tooltip = _('sets the relative weight of high local-contrast pixels. \ndefault: (0.0).'),
-  hard_min = 0,
-  hard_max = 1,
-  value = dt.preferences.read("enfuseAdvanced", "def_contrast_weight", "float")--0 default
-}
-enf_sldr_exposure_optimum_weight = dt.new_widget("slider"){
-  label = _('exposure optimum'),
-  tooltip = _('determine at what normalized exposure value\n the optimum exposure of the input images\n is. this is, set the position of the maximum\n of the exposure weight curve. use this \noption to fine-tune exposure weighting. \ndefault: (0.5)'),
-  hard_min = 0,
-  hard_max = 1,
-  value = dt.preferences.read("enfuseAdvanced", "def_exposure_optimum_weight", "float")--0.5 default0
+local AIS = {
+	name = 'align_image_stack',
+	bin = '',
+	first_run = true,
+	install_error = false,
+	arg_string = '',
+	images_string = '',
+	args = {
+		radial_distortion 		= {text = '-d', 			style = 'bool'},
+		optimize_field 			= {text = '-m', 			style = 'bool'},
+		optimize_image_center 	= {text = '-i', 			style = 'bool'},
+		auto_crop				= {text = '-C', 			style = 'bool'},
+		distortion				= {text = '--distortion', 	style = 'bool'},
+		gpu						= {text = '--gpu',			style = 'bool'},
+		grid_size				= {text = '-g ', 			style = 'integer'},
+		control_points			= {text = '-c ', 			style = 'integer'},
+		control_points_remove 	= {text = '-t ', 			style = 'integer'},
+		correlation				= {text = '--corr=', 		style = 'float'}
+	}
 }
```

Comment:
Is your editor set to convert tabs to spaces?  I'm guessing these all line up when you're editing, but when they go to a different system they look like the above.  If your editor converts tabs to spaces, then they'll always line up on any system.

Reply:
it is not, I can do so. Thanks for the heads up and sorry for the jumbled mess. Yes, it looks much cleaner in my editor :)

Reply:
I uploaded a new version converted tab to space